html/трактор.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" type="image/png" href="data:image/png;base64,">
+    <title>Трактор🚜 - Тичини</title>
+</head>
+<style>
+    /* This code is used to change the background colour of the 
+    body tag to black and the colour of the text to white 
+    when the user's computer is using dark mode. 
+    https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme
+    */
+    @media (prefers-color-scheme: dark){
+		body {color:#e3e3e3;background:#000}
+	}
+/* The font size is set to 1.2em, and the line height is set to 1.62. 
```

Comment:
format should be kept the same through all the code, at least for the specific author, if code style is not approved on the project and there not default one.